app/templates/create.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
```

Comment:
my-3 is creating the gap - but also maybe consider removing the background, and moving this header *inside* the box as per our mockup?
![Uploading image.png…]()

Reply:
(im actually unsure if my is creating the gap but i dont see what else would be. either way, you should move inside the box.)